bin/download_proteins_entrez.py

```diff
@@ -52,7 +52,7 @@ def parse_args(args=None):
         nargs="+",
         metavar="FILE",
         type=argparse.FileType("r"),
-        help="List of microbiome files containing: taxon_id [, abundance].",
+        help="List of microbiome files containing: taxon_id [,abundance].",
```

Comment:
in general the files were originally expected in TSV format but the `help` text was focussed on the content, not on the format, e.g. "file content: a, b" not containing tabs. 
The internal databases are TSV format... one could of course change this accordingly for all python scripts. Although this scripts are more for internal documentation, since the user will not directly interact with those.

Reply:
What I wanted to say is, maybe it doesn't help that much if only this help text is updated, but one should either keep it like this (not informative about the separator used), or adjust it for all python scripts. What do you think?

Reply:
This is true, and I would also suggest to make the file formats consistent over all scripts. In this case it would have helped me, as I thought for some reason there needs to be a ", " separator instead of ",". Of course this was solved by just testing and trying different versions. But in combination with a documentation in which the input types are described and shown with examples I think this could be just informative.

Reply:
So my suggestion would be to change it back to what it was and try to be specific when we write the documentation.